Cargo.toml

```diff
@@ -1,7 +1,17 @@
 [package]
-name = "derive-restricted"
+name = "derive-where"
 version = "0.1.0"
 edition = "2021"
+# rust-version
+description = "Deriving with custom trait bounds"
+documentation = "https://docs.rs/derive-where"
+readme = "README.md"
+homepage = "https://github.com/ModProg/derive-where"
```

Comment:
These should/can be removed:
> If no URL is specified in the manifest file, crates.io will automatically link your crate to the corresponding docs.rs page.

https://doc.rust-lang.org/cargo/reference/manifest.html#the-documentation-field
> If no value is specified for this field, and a file named `README.md`, `README.txt` or `README` exists in the package root, then the name of that file will be used.

https://doc.rust-lang.org/cargo/reference/manifest.html#the-readme-field

We don't have a homepage, setting it to the repository is probably not helpful. The repository link is already shown on crates.io, I think this should be enough.

What do you think?

Reply:
I agree on the homepage url, I've seen the other fields set usually even tho they are redundant, I think it can be nice to have them set anyway, for non crates.io usage.

Reply:
Good point!